layers/b2qt/recipes-qt/automotive/qtapplicationmanager-sc.inc

```diff
@@ -4,7 +4,10 @@
 #
 
 RDEPENDS_${PN} += "${PN}-softwarecontainer"
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# Due to immediate expansion of variables ${PN} is not yet available in this
+# .inc-file
```

Comment:
This comment is a bit misleading, the ${PN} variable is used just a couple of lines above. Better rephrase the comment, to mention that it's about the ":=" assignment. (but then, can't we just use "=" instead?)

Reply:
Good point! I will rephrase. No, we need to use := for `${THISDIR}` to work, I believe. I'll check though!

Reply:
Yeah, = doesn't expand properly. We need to use `:=`.

Reply:
It is pretty conventional to use either ${PN}, or "files" as the name of the directory containing extra files for the recipe. I will rename the directory to "files", and thus avoid having to hardcode "qtapplicationmanager" into the inc file.